include/Core/Posts/Modules/DeletePostsByCommentsModule.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Posts\Modules;
+
+use BulkWP\BulkDelete\Core\Posts\PostsModule;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Delete Posts by Comments Module.
+ *
+ * @since 6.1.0
+ */
+class DeletePostsByCommentsModule extends PostsModule {
+	/**
+	 * Base parameters setup.
+	 */
+	protected function initialize() {
+		$this->item_type     = 'posts';
+		$this->field_slug    = 'comments';
+		$this->meta_box_slug = 'bd_by_comments';
+		$this->action        = 'delete_posts_by_comments';
+		$this->cron_hook     = 'do-bulk-delete-comments';
+		$this->scheduler_url = 'http://bulkwp.com/addons/scheduler-for-deleting-posts-by-comments/?utm_source=wpadmin&utm_campaign=BulkDelete&utm_medium=buynow&utm_content=bd-st';
+		$this->messages      = array(
+			'box_label'  => __( 'By Post Comments', 'bulk-delete' ),
```

Comment:
Let's change this label to `By Comment count`.

Reply:
Fixed.